layers/meta-resin-qemu/recipes-core/images/resin-image.bbappend

```diff
@@ -1,5 +1,18 @@
+# Install grub
+BURN_GRUB_qemuall = "1"
```

Comment:
should this be changed here since grub install will be handled instead in this file directly?